src/main/java/org/commcare/util/LogTypes.java

```diff
@@ -113,6 +113,7 @@ public class LogTypes {
     * Logs related to Firebase Cloud Messaging
     */
     public static final String TYPE_FCM = "fcm";
-    public static final String TYPE_MEMINFO = "memory-info";
     public static final String TYPE_MEDIA_EVENT = "media-event";
+
+    public static final String TYPE_LIFECYCLE_EVENT = "lifecycle-event";
```

Comment:
don't think it serves a solid use-case to have a different type and you can just log these logs ias `TYPE_MAINTENANCE` instead.